src/app/modules/design-patterns/observer/components/roulette-bet/roulette-bet.component.ts

```diff
@@ -0,0 +1,50 @@
+import { Component, Input, OnDestroy, OnInit } from '@angular/core';
+import { filter } from 'rxjs/operators';
+
+import { RoulettetBetModel } from '../../models/roulette-bet.model';
+import { RouletteService } from '../../services/roulette.service';
+
+
+@Component({
+  selector: 'app-roulette-bet',
+  templateUrl: './roulette-bet.component.html',
+  styleUrls: ['./roulette-bet.component.scss']
+})
+export class RouletteBetComponent implements OnInit, OnDestroy {
+  bets: RoulettetBetModel[] = [];
+
+  @Input()
+  color: string;
+
+  // TODO: Add type
+  rouletteSubscription$;
+
+  constructor(private readonly rouletteService: RouletteService) {}
+
+  ngOnInit() {
+    this.betsList();
+  }
+
+  ngOnDestroy() {
+    this.rouletteSubscription$.unsubscribe();
+  }
+
+  betsList() {
+    this.rouletteSubscription$ = this.rouletteService.bets
+      .pipe(filter((bet: RoulettetBetModel) => this.isSameColor(bet)))
+      .subscribe((bet: RoulettetBetModel) => {
+        console.log(bet);
```

Comment:
Remove console.log

Reply:
remove brackets of arrow function from subscribe and put `this.pushBetToList(bet)`